docs/snippets/ov_extensions.cpp

```diff
@@ -55,6 +56,7 @@ class CustomOp : public ov::op::Op {
     OPENVINO_OP("CustomOp");
     OPENVINO_FRAMEWORK_MAP(onnx, "CustomOp", { {"m_mode", "mode"} }, { {"m_axis", -1} });
     OPENVINO_FRAMEWORK_MAP(tensorflow, "CustomOpV3", { {"m_axis", "axis"} }, { {"m_mode", "linear"} });
+    OPENVINO_FRAMEWORK_MAP(paddle, {"X"}, {"Out"}, "CustomOp", { {"m_mode", "mode"} }, { {"m_axis", -1} });
```

Comment:
Using `m_` prefix in an OV attribute name looks unnatural. In other examples (e.g. `CustomElu`) we don't use this prefix for attribute names (only in the name of corresponding operation class field that holds a value of the attribute). I mean it leads to kind of confusion: why not to name attribute without `m_` and define less complex (but still not trivial -- this is important for education) mapping rules. This is a suggestion for the future improvements.

Reply:
done